src/reporting/feedback.tsx

```diff
@@ -131,85 +121,34 @@ function reducer(state: Event, { type, payload }: { type: string; payload: any }
 	}
 }
 
-const getTopics = (t: TFunction): Array<{ label: string; value: string }> => [
-	{ label: t('feedback.user_interface', 'User interface'), value: 'UserInterface' },
-	{ label: t('feedback.behaviors', 'Behaviors'), value: 'Behaviors' },
-	{ label: t('feedback.missing_features', 'Missing features'), value: 'MissingFeatures' },
-	{ label: t('feedback.other', 'Other'), value: 'Other' }
-];
-
 const Feedback: FC = () => {
 	const [t] = useTranslation();
-	const topics = useMemo(() => getTopics(t), [t]);
 	const allApps = useAppList();
-	const [feedbackPermission, setFeedbackPermission] = useState(false);
-	const [feedbackSentData, setFeedbackSentData] = useState('');
+	const feedbackPermission = useConfigStore(
+		(state) =>
+			state.getConfigAttribute('carbonioSendFullErrorStack') === 'TRUE' &&
+			state.getConfigAttribute('carbonioSendAnalytics') === 'TRUE' &&
+			state.getConfigAttribute('carbonioAllowFeedback') === 'TRUE'
```

Comment:
Please use TRUE from constant file . Instead of write static keyword 'TRUE'

Reply:
Resolved please check